src/android/nl/xservices/plugins/Calendar.java

```diff
@@ -694,18 +696,21 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
       if (resultCode == Activity.RESULT_OK || resultCode == Activity.RESULT_CANCELED) {
         // resultCode may be 0 (RESULT_CANCELED) even when it was created, so passing nothing is the clearest option here
         Log.d(LOG_TAG, "onActivityResult resultcode: " + resultCode);
-        callback.success();
+        if (callback != null)
```

Comment:
Curiosity: does upstream suffer from the same & have they dealt with it?

Reply:
Upstream is not changed. It is a very unlikely crash since the memory used by the calendar activities is low.

If the webview activity is recreated, then the plugins are recreated as well, making the instance variable callback have a null value because it is never assigned with anything. When onActivityResult is called the app crashes with a null pointer exception.

Reply:
This could be a good opportunity to contribute to upstream such a tactical fix, or at least raise awareness to the issue (although I did not check upstream's issues) 🤔